meta/netbeansrelease.json

```diff
@@ -441,6 +441,34 @@
             "year": "2021"
         }
     },
+    "release1243": {
+        "position": "12",
+        "ant": "ant_latest",
+        "jdk": "jdk_1.8_latest",
+        "jdk_apidoc": "https://docs.oracle.com/javase/8/docs/api/",
+        "maven": "maven_3.3.9",
+        "versionName": "12.4.301",
+        "tlp": "true",
+        "apidocurl": "https://bits.netbeans.org/12.4/javadoc",
```

Comment:
@ebarboni can this be left out? Will this all cause an overwrite of existing 12.4 Javadocs??

Reply:
we may use  vsnetbeans_preview_1243 as key and triage differently on TLP build ? Otherwise it will mess up the javadoc

Reply:
I've changed the location of the Javadoc in 71fcead, but I am more and more inclined towards avoiding the TLP build run completely. The only reason for running it is to get proper sources ZIP file. However such ZIP file may be generated by other jobs as well.

I've just modified the [netbeans-vscode job](https://ci-builds.apache.org/job/Netbeans/job/netbeans-vscode/) to execute `ant build-source-config` first. Builds since 493 shall always expose the source ZIP as another build artifact. I guess I can call the vote over that ZIP and its complementary `.vsix` binary.